Release v0.1.0 (#16)
camaraproject/NumberRecycling: CHANGELOG.md

# Number Recycling

## Table of Contents

- [v0.1.0](#v010)

**Please be aware that the project will have frequent updates to the main branch. There are no compatibility guarantees associated with code in any branch, including main, until it has been released. For example, changes may be reverted before a release is published. For the best results, use the latest published release.**

> # v0.1.0

Reviewer: ```suggestion
# r1.1
```

> **This is the first alpha version of the KYC APIs.** 

Reviewer 2: ```suggestion
**This is the first alpha version of the Number Recycling API.** 
```

> - API [definition](https://github.com/camaraproject/NumberRecycling/tree/main/code/API_definitions)

Reviewer: Link should be to release tag (r1.1) rather than main branch
```suggestion
- [number-recycling v0.1.0-alpha.1](https://github.com/camaraproject/NumberRecycling/blob/r1.1/code/API_definitions/number-recycling.yaml)
```

> - [number-recycling v0.1.0-alpha.1](https://github.com/camaraproject/NumberRecycling/blob/r1.1/code/API_definitions/number-recycling.yaml)

Reviewer 2: Here is no link expected, the links are in the section for the number-recycling v0.1.0-alpha.1 API.

See https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/CHANGELOG_TEMPLATE.md
for the template.

Author: Thanks @hdamker 
CHANGELOG.md was updated based on the template.

Reviewer 2: @yamamoto0104 the review issue https://github.com/camaraproject/ReleaseManagement/issues/133 is actually assigned to @FrimanJan. And don't worry, the deadline only requires to have the PR provided until Friday for review by ReleaseManagement. That you have achieved 👍

* Note: API documentation is embedded in the YAML files.

## Please note:

- This is an alpha version, it should be considered as a draft.

## What's changed

> * New API `Number Recycling`, v0.1.0:

Reviewer: ```suggestion
* New API `Number Recycling`, v0.1.0-alpha.1:
```
